presto-main/src/main/java/com/facebook/presto/operator/aggregation/TDigestAggregationFunction.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Licensed to Ted Dunning under one or more
```

Comment:
If this gets squashed then it should be removed from the commit history entirely